### ISSUE TITLE
inlinecontent save button click event is not firing asp.net

### ISSUE THREAD

**Issue by skhurams** [nova706/OrangeBox#3]
hi orangebox is very easy but i got  a problem i have  a button in inlinecontent of orangebox
here is my code 

```
                           <span class="pull-right"><a data-ob_share="false"   href="#inlinecontent" rel="lightbox" style="color: white">Edit your profile</a></span>
```

<div id="inlinecontent" style="display: none;">
             
                      <div class="row-fluid" style="padding-left: 10px; background-color: #f2f3f4;">
                    <span class="span5" style="text-align: right;">Full Name </span><span class="span7">
                        <asp:TextBox ID="txtName" runat="server" Text="Full Name"></asp:TextBox></span>
                </div>
                

```
                                <div class="row-fluid" style="padding-left: 10px; background-color: #f2f3f4;">
                                      <span class="span5"></span><span class="span7">
                                        <asp:Button runat="server" CssClass="btn btn-primary" ID="btnSave" Text="Save" OnClick="btnSave_Click" />
                                     <asp:Button runat="server" CssClass="btn" ID="btnClose" Text="Cancel" />
                </span>
            </div>
        </div>
```

when i click save button nothing happens and no postback occurs im not using scriptmanager or ajax. One other thing i want to close lightbox on clicking btnClose
